webpack/components/Table/components/TranslatedPlural.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+// a defaultMessage like
+// "{count, plural, =0 {No errata} one {# erratum} other {# errata}}"
+// will give us properly translated pluralized strings!
+// see https://formatjs.io/docs/react-intl/components/#message-syntax
+export const TranslatedPlural = ({
```

Comment:
I'm curious why we don't use [pluralize](https://apidock.com/rails/ActionView/Helpers/TextHelper/pluralize) method directly.

Reply:
That's a back-end method; it wouldn't help us here..